examples/window-app/common/window-app.zap

```diff
@@ -32,8 +32,8 @@
   "endpointTypes": [
     {
       "name": "MA-windowcovering",
-      "deviceTypeName": "MA-windowcovering",
-      "deviceTypeCode": 514,
+      "deviceTypeName": "MA-rootdevice",
```

Comment:
i never understood why there is this prefix MA ?

Reply:
@jmeg-sfy I believe that is just the prefix that was chose for Matter devices in the metadata. Similar to HA -> "Home Automation", ZLL -> "Zigbee Light Link", SE -> "Smart Energy" and so on..